webpack/scenes/Subscriptions/Details/SubscriptionDetails.js

```diff
@@ -45,7 +58,7 @@ class SubscriptionDetails extends Component {
     }
 
     return (
-      <Grid bsClass="container-fluid">
+      <div>
```

Comment:
Why this change?

Reply:
This Grid moved to inside the TabContainer, on [line 91](https://github.com/Katello/katello/pull/7578/files#diff-d67a0043161dfe1a4d75f6f64d995cbfR91).